TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -48,15 +66,24 @@ def __init__(self, config: Dict) -> None:
         self.endpoint = config["endpoint"]
         if self.endpoint.startswith("https"):
             self.allow_insecure = config.get("allow_insecure", False)
-        if "authorization" in config:
+        try:
             self.auth_type = AuthMethod[config["authorization"]["type"].upper()]
+        except KeyError:
+            self.auth_type = None
+        try:
             self.auth_details = config["authorization"]["details"]
+        except KeyError:
+            self.auth_details = None
```

Comment:
in general, I think `self.auth_details = config["authorization"].get("details", None)` is a little more concise and clearer. I'd do `try/except` in this case if we intended to do a more complicated behavior or throw an error if it were missing.

not a blocker though.

Reply:
Definitely cleaner, updated